app_feup/lib/view/Pages/LoginPageView.dart

```diff
@@ -32,39 +34,41 @@ class LoginPageView extends StatelessWidget {
     return Scaffold(
         backgroundColor: primaryColor,
         resizeToAvoidBottomPadding: false,
-        body: Center(
-          child: Padding(
-              padding: EdgeInsets.only(
-                  left: queryData.size.width / 8,
-                  right: queryData.size.width / 8,
-                  top: queryData.size.height / 6,
-                  bottom: queryData.size.height / 6),
-              child: Flex(
-                direction: Axis.vertical,
-                children: <Widget>[
-                  createTitle(queryData),
-                  Spacer(),
-                  Form(
-                    key: this.formKey,
-                    child: Column(children: [
-                      createUsernameInput(context),
+        body: HomePageBackButton(
```

Comment:
Maybe this HomePageBackButton should now have a different name

Reply:
Any suggestion for the name?

Reply:
Probably something like BackButtonExitWrapper or ExitOnBackButton